.gitignore

```diff
@@ -0,0 +1,2 @@
+
+.idea
```

Comment:
Why add this file?
We already have this file https://github.com/Swati4star/Travel-Mate/blob/master/Android/.gitignore